WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewModel.swift

```diff
@@ -86,6 +118,12 @@ final class InPersonPaymentsMenuViewModel {
             self?.showWebView = nil
         })
     }
+
+    deinit {
+        NotificationCenter.default.removeObserver(self,
```

Comment:
Yeah, the way it was written in the docs suggested it was still good practice, so I did it anyway because it's trivial...

> If your app targets iOS 9.0 and later or macOS 10.11 and later, and you used [addObserver(_:selector:name:object:)](dash-apple-api://load?request_key=ls/documentation/foundation/notificationcenter/1415360-addobserver) to create your observer, you do not need to unregister the observer. If you forget or are unable to remove the observer, the system cleans up the next time it would have posted to it.